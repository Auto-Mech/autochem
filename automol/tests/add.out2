2
2
reaction class: addition
forward TS atoms:
  1: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  2: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  3: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  4: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  5: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  6: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  7: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  8: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  9: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  10: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  11: {symbol: O, implicit_hydrogen_valence: 0, stereo_parity: null}
  12: {symbol: O, implicit_hydrogen_valence: 0, stereo_parity: null}
forward TS bonds:
  1-2: {order: 1, stereo_parity: null}
  1-3: {order: 1, stereo_parity: null}
  1-4: {order: 1, stereo_parity: null}
  1-5: {order: 1, stereo_parity: null}
  2-6: {order: 1, stereo_parity: null}
  2-7: {order: 1, stereo_parity: null}
  2-8: {order: 1, stereo_parity: null}
  6-9: {order: 1, stereo_parity: null}
  6-10: {order: 1, stereo_parity: null}
  6-11: {order: 0.1, stereo_parity: null}
  11-12: {order: 1, stereo_parity: null}
reactants keys:
- [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
- [11, 12]
backward TS atoms:
  1: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  2: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  3: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  4: {symbol: O, implicit_hydrogen_valence: 0, stereo_parity: null}
  5: {symbol: O, implicit_hydrogen_valence: 0, stereo_parity: null}
  6: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  7: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  8: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  9: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  10: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  11: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  12: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
backward TS bonds:
  1-2: {order: 1, stereo_parity: null}
  1-6: {order: 1, stereo_parity: null}
  1-7: {order: 1, stereo_parity: null}
  1-8: {order: 1, stereo_parity: null}
  2-3: {order: 1, stereo_parity: null}
  2-9: {order: 1, stereo_parity: null}
  2-10: {order: 1, stereo_parity: null}
  3-5: {order: 0.9, stereo_parity: null}
  3-11: {order: 1, stereo_parity: null}
  3-12: {order: 1, stereo_parity: null}
  4-5: {order: 1, stereo_parity: null}
products keys:
- [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]

C  
C  0    R1 
H  0    R2  1    A2 
H  0    R3  1    A3  2    D3 
H  0    R4  1    A4  2    D4 
C  1    R5  0    A5  2    D5 
H  1    R6  0    A6  5    D6 
H  1    R7  0    A7  5    D7 
H  5    R8  1    A8  0    D8 
H  5    R9  1    A9  8    D9 
O  5   R10  1   A10  8   D10 
O  10   R11  5   A11  1   D11 

R1   =   1.518515
R2   =   1.095355
R3   =   1.094712
R4   =   1.095355
R5   =   1.489685
R6   =   1.095253
R7   =   1.095253
R8   =   1.083842
R9   =   1.083842
R10  =   1.900000
R11  =   1.153017
A2   = 111.217315
A3   = 110.220672
A4   = 111.217338
A5   = 111.518495
A6   = 109.804533
A7   = 109.804497
A8   = 120.268634
A9   = 120.268653
A10  =  85.000000
A11  =  85.000000
D3   = 240.274937
D4   = 120.549941
D5   = 299.724886
D6   = 238.803526
D7   = 121.196400
D8   = 273.502299
D9   = 172.998503
D10  = 266.497733
D11  =  85.000000
{'D8', 'D5', 'D11'}
R10
()
C    0.000000   0.000000   0.000000
C    0.000000   0.000000   1.518515
H    0.000000   1.021105  -0.396416
H   -0.892073  -0.509347  -0.378373
H    0.879362  -0.519017  -0.396416
C    1.203497   0.687156   2.064934
H   -0.026464  -1.030133   1.889600
H   -0.900585   0.500821   1.889600
H    2.126351   0.138166   2.212125
H    1.199780   1.760944   2.212173
O    0.466799   0.266525   3.765036
O    0.995338  -0.749336   3.630418
axis: [0, 1]
	group 1: (2, 3, 4)
	group 2: (5, 6, 7, 8, 9, 10, 11)
	symmetry number: 3
axis: [1, 5]
	group 1: (0, 2, 3, 4, 6, 7)
	group 2: (8, 9, 10, 11)
	symmetry number: 1
axis: [5, 10]
	group 1: (0, 1, 2, 3, 4, 6, 7, 8, 9)
	group 2: (11,)
	symmetry number: 1
C  
C  0    R1 
H  0    R2  1    A2 
H  0    R3  1    A3  2    D3 
H  0    R4  1    A4  2    D4 
C  1    R5  0    A5  2    D5 
H  1    R6  0    A6  5    D6 
H  1    R7  0    A7  5    D7 
C  5    R8  1    A8  0    D8 
H  5    R9  1    A9  8    D9 
H  5   R10  1   A10  8   D10 
C  8   R11  5   A11  1   D11 
H  8   R12  5   A12  11   D12 
H  8   R13  5   A13  11   D13 
C  11   R14  8   A14  5   D14 
H  11   R15  8   A15  14   D15 
H  11   R16  8   A16  14   D16 
C  14   R17  11   A17  8   D17 
H  14   R18  11   A18  17   D18 
H  14   R19  11   A19  17   D19 
C  17   R20  14   A20  11   D20 
H  17   R21  14   A21  20   D21 
H  20   R22  17   A22  14   D22 
H  20   R23  17   A23  22   D23 
C  20   R24  17   A24  22   D24 
C  24   R25  20   A25  17   D25 
H  24   R26  20   A26  25   D26 
H  24   R27  20   A27  25   D27 
H  25   R28  24   A28  20   D28 
H  25   R29  24   A29  28   D29 
H  25   R30  24   A30  28   D30 

R1   =   1.520718
R2   =   1.094017
R3   =   1.094775
R4   =   1.094389
R5   =   1.530643
R6   =   1.095155
R7   =   1.096832
R8   =   1.532923
R9   =   1.097101
R10  =   1.097281
R11  =   1.531807
R12  =   1.096396
R13  =   1.095887
R14  =   1.531578
R15  =   1.096767
R16  =   1.096392
R17  =   1.508109
R18  =   1.097750
R19  =   1.097249
R20  =   1.322012
R21  =   1.081693
R22  =   1.085116
R23  =   1.108961
R24  =   2.031555
R25  =   1.486583
R26  =   1.097100
R27  =   1.119993
R28  =   1.085020
R29  =   1.101273
R30  =   1.102427
A2   = 111.370348
A3   = 111.113476
A4   = 110.112332
A5   = 113.537160
A6   = 109.611433
A7   = 108.445026
A8   = 115.768832
A9   = 108.492507
A10  = 108.730814
A11  = 113.340224
A12  = 107.535203
A13  = 110.139430
A14  = 113.171364
A15  = 109.613991
A16  = 109.419712
A17  = 113.577765
A18  = 109.033088
A19  = 108.505508
A20  = 123.892768
A21  = 117.234942
A22  = 127.704141
A23  = 121.106560
A24  =  80.811854
A25  =  87.225801
A26  =  71.684458
A27  = 153.533439
A28  = 108.869596
A29  = 116.618447
A30  = 111.857228
D3   = 121.406602
D4   = 240.854519
D5   = 293.085460
D6   = 124.022588
D7   = 239.687650
D8   =  59.274441
D9   = 122.800807
D10  = 237.825346
D11  =  61.248625
D12  = 121.279426
D13  = 236.517276
D14  = 177.781265
D15  = 237.459489
D16  = 120.323646
D17  = 293.267691
D18  = 124.664703
D19  = 240.258002
D20  = 123.168440
D21  = 178.375466
D22  = 190.420174
D23  = 163.653612
D24  = 295.970873
D25  =  84.430946
D26  = 121.107126
D27  = 216.172370
D28  = 348.223995
D29  = 121.534785
D30  = 242.427493
{'D28', 'D25', 'D11', 'D5', 'D14', 'D8', 'D17', 'D20'}
R24
()
C    0.000000   0.000000   0.000000
C    0.000000   0.000000   1.520718
H    0.000000   1.018797  -0.398654
H    0.871655  -0.532198  -0.394356
H   -0.897538  -0.500497  -0.376318
C    1.290921   0.550237   2.131970
H   -0.866252   0.560231   1.888295
H   -0.130892  -1.032219   1.867749
C    1.646777   1.991914   1.751460
H    1.217200   0.477858   3.224196
H    2.124729  -0.099972   1.838685
C    0.593482   3.013354   2.191534
H    2.608344   2.229848   2.221419
H    1.809092   2.067698   0.670313
C    0.978145   4.457709   1.857463
H    0.419418   2.921137   3.270467
H   -0.357692   2.794341   1.692157
C    2.171839   4.958439   2.631257
H    1.141077   4.543624   0.775277
H    0.128045   5.108800   2.096976
C    3.270020   5.431782   2.067644
H    2.102087   4.947638   3.710645
H    4.216081   5.661183   2.547039
H    3.410843   5.396130   0.968239
C    2.838535   6.990581   3.296944
C    3.551566   6.248290   4.369567
H    3.422938   7.370979   2.449950
H    2.183828   7.815526   3.678008
H    3.730717   5.232243   4.033676
H    4.519071   6.660946   4.695836
H    2.947650   6.174700   5.288922
axis: [0, 1]
	group 1: (2, 3, 4)
	group 2: (5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30)
	symmetry number: 3
axis: [1, 5]
	group 1: (0, 2, 3, 4, 6, 7)
	group 2: (8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30)
	symmetry number: 1
axis: [5, 8]
	group 1: (0, 1, 2, 3, 4, 6, 7, 9, 10)
	group 2: (11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30)
	symmetry number: 1
axis: [8, 11]
	group 1: (0, 1, 2, 3, 4, 5, 6, 7, 9, 10, 12, 13)
	group 2: (14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30)
	symmetry number: 1
axis: [11, 14]
	group 1: (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 15, 16)
	group 2: (17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30)
	symmetry number: 1
axis: [14, 17]
	group 1: (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 15, 16, 18, 19)
	group 2: (20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30)
	symmetry number: 1
axis: [20, 24]
	group 1: (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 22, 23)
	group 2: (25, 26, 27, 28, 29, 30)
	symmetry number: 1
axis: [24, 25]
	group 1: (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 26, 27)
	group 2: (28, 29, 30)
	symmetry number: 3
