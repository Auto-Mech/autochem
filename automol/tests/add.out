reaction class: addition
forward TS atoms:
  1: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  2: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  3: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  4: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  5: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  6: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  7: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  8: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  9: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  10: {symbol: O, implicit_hydrogen_valence: 0, stereo_parity: null}
  11: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
forward TS bonds:
  1-2: {order: 1, stereo_parity: null}
  1-3: {order: 1, stereo_parity: null}
  1-4: {order: 1, stereo_parity: null}
  1-5: {order: 1, stereo_parity: null}
  2-6: {order: 1, stereo_parity: null}
  2-7: {order: 1, stereo_parity: null}
  6-8: {order: 1, stereo_parity: null}
  6-9: {order: 1, stereo_parity: null}
  6-10: {order: 0.1, stereo_parity: null}
  10-11: {order: 1, stereo_parity: null}
reactants keys:
- [1, 2, 3, 4, 5, 6, 7, 8, 9]
- [10, 11]
backward TS atoms:
  1: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  2: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  3: {symbol: C, implicit_hydrogen_valence: 0, stereo_parity: null}
  4: {symbol: O, implicit_hydrogen_valence: 0, stereo_parity: null}
  5: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  6: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  7: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  8: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  9: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  10: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
  11: {symbol: H, implicit_hydrogen_valence: 0, stereo_parity: null}
backward TS bonds:
  1-2: {order: 1, stereo_parity: null}
  1-5: {order: 1, stereo_parity: null}
  1-6: {order: 1, stereo_parity: null}
  1-7: {order: 1, stereo_parity: null}
  2-3: {order: 1, stereo_parity: null}
  2-8: {order: 1, stereo_parity: null}
  3-4: {order: 0.9, stereo_parity: null}
  3-9: {order: 1, stereo_parity: null}
  3-10: {order: 1, stereo_parity: null}
  4-11: {order: 1, stereo_parity: null}
products keys:
- [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]

